.github/workflows/cae.yml

```diff
@@ -0,0 +1,65 @@
+name: CAE Build/Release
+
+on:
+  push:
+    # takes muliple branch names
+    branches: 
+      - main
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.x
+    tags:
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9][0-9]' # 2021.01.01
+ 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+        default: 'YYYY.MINOR.MICRO'
+env:
+  REPO_URL: ${{ github.repository }}
+    
+jobs:
+  build:
+    # to test a feature, change the repo name to your github id
+    if: github.repository_owner == 'tl-its-umich-edu' || github.event_name == 'workflow_dispatch'
```

Comment:
I am pointing to TL repo only since I did not want to forked repo for a build ( this step is common to all TL Repos)

`workflow_dispatch` will let us do a manual trigger of the build